synapse/lib/spooled.py

```diff
@@ -15,7 +17,7 @@ class Spooled(s_base.Base):
     together. Under memory pressure, these objects have a better shot of getting paged out.
     '''
 
-    async def __anit__(self, dirn=None, size=10000, cell=None):
+    async def __anit__(self, dirn=None, size=0, cell=None):
```

Comment:
If this value is going to be set to a default, that should be declared here rather than in the runtime below.

Reply:
Addressed per goback.